xml/System.Windows.Forms/SplitterPanel.xml

```diff
@@ -175,14 +161,7 @@
       </ReturnValue>
       <Docs>
         <summary>This event is not relevant to this class.</summary>
```

Comment:
This event summary looks sad. Should it be prefixed with a small description?

Reply:
I'll add one.